File: sort_associative_array_by_value_ascend/sort_associative_array_by_value_ascend.pl
use strict;
my $example_array = {'January' => 1, 'December' => 12, 'March' => 3 };
foreach my $key (sort {$example_array->{$a} <=> $example_array->{$b}} keys %{$example_array}) {
   print $key. " => " . $example_array->{$key} ."\n";
}

File: sort_associative_array_by_value_ascend/sort_associative_array_by_value_ascend.js
let example_array = new Map( [ ["January", 1], ["December", 12], ["March", 3] ].sort((a, b) => {a = a[1]; b = b[1]; return a - b;}) );
for (let [key, value] of example_array.entries()) {
  console.log(key + ' => ' + value);
}
